src/thunderbolt/device.c

```diff
@@ -0,0 +1,384 @@
+#include "thunderbolt.h"
```

Comment:
Normally we'd order the includes for all internal headers in one big block alphabetically.